mpyg321/mpyg321.py

```diff
@@ -173,6 +174,14 @@ def process_output(self):
             if action == "user_silence":
                 self.on_silence()
 
+    def build_options(self, kwoptions):
+        """Build options"""
+        options = ["--remote"]
```

Comment:
Linking this to issue #15 - is it retrocompatible with mpg321?
Also, is there a way to detect easily which version is running so that we can adapt the code to it?